### PR TITLE
[i18n] Add missing Russian strings

### LINKED ISSUE #2554: [i18n] Improve Russian language
### Description:

Russian translation have a lot of missing strings.
I have a fix for that

### PR DESCRIPTION
**Pull request by ankar84**
## Proposed changes
Russian translation was missing a lot of strings, this PR adds all the missing ones and fixes some of the existing strings.

## Issue(s)	
closes #2554

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ankar84** [comment]
**Карлан Антон Андреевич** it's me actually pulled commits with unset properly  git client on my machine. I don't know how get rid of that now.

**Comment by ankar84** [comment]
@Pitstopper can you take a look on proposed changes?
I see, that you [worked](https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/381) on Russian locale in that repo.
Thanks!

File: app/i18n/locales/ru.js [hunk 1]
```diff
@@ -139,41 +158,76 @@ export default {
...
 	Create_account: 'Создать аккаунт',
 	Create_Channel: 'Создать канал',
+	Create_Direct_Messages: 'Создать Личное сообщение',
+	Create_Discussion: 'Создать Обсуждение',
 	Created_snippet: 'Создать сниппет',
 	Create_a_new_workspace: 'Новое рабочее пространство',
 	Create: 'Создать',
+	Custom_Status: 'Персонализированный Статус',
+	Dark: 'Темная',
+	Dark_level: 'Уровень Темной Темы',
```
Comment (Pitstopper) [inline app/i18n/locales/ru.js:176]: Maybe - "Степень затемнения" или "Уровень затемненности"?

File: app/i18n/locales/ru.js [hunk 2]
```diff
@@ -185,35 +239,69 @@ export default {
...
+	IP: 'IP',
+	In_app: 'В-приложении',
 	IN_APP_AND_DESKTOP: 'В приложении и на десктопе',
 	In_App_and_Desktop_Alert_info: 'Отображает баннер в верхней части экрана, когда приложение открыто, и отображает уведомление на рабочем столе.',
 	Invisible: 'Невидимый',
 	Invite: 'Приглашение',
 	is_a_valid_RocketChat_instance: 'является действительным сервером Rocket.Chat',
 	is_not_a_valid_RocketChat_instance: 'не является действительным сервером Rocket.Chat',
 	is_typing: 'печатает',
+	Invalid_or_expired_invite_token: 'Токен приглашение не валидный или с истекшим сроком действия',
```
Comment (Pitstopper) [inline app/i18n/locales/ru.js:280]: Maybe - "Токен приглашения не действителен или с истекшим сроком действия"?

File: app/i18n/locales/ru.js [hunk 3]
```diff
@@ -411,27 +561,103 @@ export default {
 	Username_is_empty: 'Имя пользователя пусто',
 	Username: 'Имя пользователя',
 	Username_or_email: 'Имя пользователя или email',
+	Uses_server_configuration: 'Uses server configuration',
```
Comment (Pitstopper) [inline app/i18n/locales/ru.js:564]: Translate this please! (If it needs to be translated...)

File: app/i18n/locales/ru.js [hunk 4]
```diff
@@ -411,27 +561,103 @@ export default {
...
+	New_line: 'Новая линия',
+	You_will_be_logged_out_of_this_application: 'Будет осуществлен ваш выход из этого приложения.',
+	Clear: 'Очистка',
+	This_will_clear_all_your_offline_data: 'Это очистит все ваши офлайн данные.',
+	This_will_remove_all_data_from_this_server: 'Это удалит все данные с этого сервера.',
+	Mark_unread: 'Отметить Непрочитанным',
+	Wait_activation_warning: 'До того как вы сможете войти, ваш аккаунт должен быть вручную активирован администратором сервера.',
+	Screen_lock: 'Блокировка экрана',
+	Local_authentication_biometry_title: 'Аутентификация',
+	Local_authentication_biometry_fallback: 'Использовать паскод',
```
Comment (Pitstopper) [inline app/i18n/locales/ru.js:635]: Maybe - пасскод or пасс-код?

**Review by Pitstopper — CHANGES_REQUESTED** [review]
Hey! You have done a great job! My respect!  But there are some points that need to be fixed for better display in the Russian version of the rocketchat client.

File: app/i18n/locales/ru.js [hunk 5]
```diff
@@ -139,41 +158,76 @@ export default {
...
 	Copy: 'Копировать',
+	Conversation: 'Диалог',
 	Permalink: 'Постоянная ссылка',
 	Certificate_password: 'Пароль сертификата',
+	Clear_cache: 'Очистить локальный кэш сервера',
+	Clear_cache_loading: 'Очистка кэша.',
 	Whats_the_password_for_your_certificate: 'Какой пароль для вашего сертификата?',
 	Create_account: 'Создать аккаунт',
 	Create_Channel: 'Создать канал',
+	Create_Direct_Messages: 'Создать Личное сообщение',
```
[the remark this answers is not in the document]
Comment (ankar84) [reply app/i18n/locales/ru.js:169]: Agree with you. I just try to translate with accurate accordance to English variant - where all was capitalized, I did same in Russian.
But yes, I will fix that, thanks!

File: app/i18n/locales/ru.js [hunk 6]
```diff
@@ -259,20 +354,40 @@ export default {
...
+	Onboarding_more_options: 'Больше опций',
 	Online: 'Онлайн',
 	Only_authorized_users_can_write_new_messages: 'Только авторизованные пользователи могут писать новые сообщения',
-	Open_emoji_selector: 'Открыть селектор emoji',
+	Open_emoji_selector: 'Открыть выбор emoji',
 	Open_Source_Communication: 'Общение с открытым кодом',
+	Open_your_authentication_app_and_enter_the_code: 'Откройте ваше приложение для аутентификации и введите код и него.',
+	OR: 'OR',
+	OS: 'OS',
+	Overwrites_the_server_configuration_and_use_room_config: 'Overwrites the server configuration and use room config',
```
[the remark this answers is not in the document]
Comment (ankar84) [reply app/i18n/locales/ru.js:378]: Sure, I will definitely translate `Overwrites_the_server_configuration_and_use_room_config: Overwrites the server configuration and use room config,` in next commit
But I want to ask @diegolmello where is OR and OS is applied? Is that OS is for Operating System or something else? OR is like programmers `||` or what?

[the remark this answers is not in the document]
Comment (diegolmello) [reply app/i18n/locales/ru.js:378]: - `OR` is literal and used with a separator on login.

```
login with services

OR

login with user/password
```

- `OS` is `Operating System` and is used on Omnichannel.

File: app/i18n/locales/ru.js (hunk 4 above)
Comment (ankar84) [reply app/i18n/locales/ru.js:635]: I though maybe simple `пароль`?
I did a `паскод` translate for `passcode` to make some difference between regular user password, but simple translate for `passcode` is `пароль`.
What do you think?

Comment (Pitstopper) [reply app/i18n/locales/ru.js:635]: Sounds good!

**Review by Pitstopper — APPROVED** [review]
Well done :+1:

**Comment by diegolmello** [comment]
@ankar84 can you update?

**Comment by ankar84** [comment]
> @ankar84 can you update?

@diegolmello I don't know how to do it now actually.
I think is something like:
1. Somehow fork again https://github.com/RocketChat/Rocket.Chat.ReactNative/ or somehow clone or something to my repo
2. Compare that refreshed up-to-date repo to my russian-strings branch and translate new missing strings.

But the problem is I really don't know actual steps (commands) to do that.
For me it's better to merge that PR and then I start from scratch and translate new missing strings.

**Comment by ankar84** [comment]
Oh, I think I broke everything when try to resolve conflicts.
So, let that PR lost in history...

**Review by diegolmello — COMMENTED** [review]
@ankar84 I've added missing `en` strings on this commit: https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/2555/commits/0ddd9db7fa8851f414d7054ecf4d7c59a36b0320

Can you translate them and send me on DM (then I can add them here)?
Thanks!

**Comment by ankar84** [comment]
> @ankar84 I've added missing `en` strings on this commit: [0ddd9db](https://github.com/RocketChat/Rocket.Chat.ReactNative/commit/0ddd9db7fa8851f414d7054ecf4d7c59a36b0320)
> 
> Can you translate them and send me on DM (then I can add them here)?
> Thanks!

@diegolmello done with your help.

**Review by diegolmello — APPROVED** [review]
LGTM

**Pull request merged**